axes/tests/settings.py

```diff
@@ -7,7 +10,11 @@
 
 CACHES = {
     'default': {
-        'BACKEND': 'django.core.cache.backends.dummy.DummyCache'
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
```

Comment:
@aleksihakli I'm wondering why it would be better to have a file and not a dummy cache for the tests?

Reply:
The dummy cache doesn't work in the cache tests since it doesn't record any of the failed attempts whereas the FileBasedCache was easy enough to configure for Travis compatibility. It is also suited for a single server deployments on Axes, so I thought it would be a good default in the test bench.

Reply:
@aleksihakli right, right :)